src/transformers/pipelines/base.py

```diff
@@ -1010,7 +1037,6 @@ def __init__(
             self.device = self.model.device
         logger.warning(f"Device set to use {self.device}")
 
-        self.binary_output = binary_output
```

Comment:
(let's not delete this, unless we're deleting all `binary_output`-related references. It seems unused, yes)